tools/test_storage_hyperv_primitive.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+from cloudcix_primitives import storage_hyperv
+
+cmd = sys.argv[1]
+
+host = None
+domain_path = 'D:\HyperV\primitive_test\\'
+storage = 'primitive_test.vhdx'
+size = 5
+update_size = 6
```

Comment:
```suggestion
```

It should be  possible to use the `size` variable in both build and update queries by changing the value in the test call.